localScrapper.js

```diff
@@ -8,9 +8,14 @@ const isVideoExtension = extension => [
 function getFileInfos(fileName, dirPath, stat) {
   const lastIndexOfSlash = dirPath.lastIndexOf('/')
   const dirName = lastIndexOfSlash !== -1 ? dirPath.substr(lastIndexOfSlash + 1) : '.'
-  const fileNamePatterns = fileName.match(/^(.+) - ([A-Z0-9 ]+) - (.+)\.(.{2,4})$/) || []
-  const [, group, type, songName, extension] = fileNamePatterns
+  const fileNamePatterns = fileName.match(/^(.+) - ([A-Z0-9 ]+) - (.+) ?(\(.+\))?\.(.{2,4})$/) || []
```

Comment:
Two problems with that regex.

The pattern `(.+)` before the language catch is greedy, so it'll always catch the more it can.
That can be solved with using this pattern: 

(/^(.+) - ([A-Z0-9 ]+) - (.+?) ?(\(.+\))?\.(.{2,4})$/`

Notice the `.+?`.

But that creates another problem with some files like this one: "Kamen Rider W - OP - W-B-X (W-Boiled Extreme).mkv".
Even though these files are located in my TEMP directory so they should not be named thusly.